db/seeds.rb

```diff
@@ -27,6 +29,22 @@
 
 puts ">>> SEEDING ENVIRONMENT: #{Rails.env}"
 
+if Region.all.empty?
```

Comment:
The source for blank? is:

def blank?
  respond_to?(:empty?) ? !!empty? : !self
end

So it doesn't help much here. Calling it on the activerecord class would always result in false, and calling it on .all would be the same as empty?